react-native/architecture/project-architecture.s.md

```diff
@@ -2,11 +2,44 @@
 
 ## Checks
 - Entire code except `index.ios.js` and `index.android.js` should be in `src`
-- The structure should be as follow
+- The structure should be as follow and follow the linked standards
   - `index.ios.js`
   - `index.android.js`
   - `/src`
-    - `modules`
-    - `components`
-    - `pages`
-    - `style`
+    - `package.json`
+    - `App.js`
+    - [`/components`](./components/folder.s.md)
+      - [`index.js`](./components/index.s.md)
+      - `package.json`
+      - [`/MyComponent`](./components/my-component/folder.s.md)
+        - [`MyComponent.component.js`](./components/my-component/component.s.md)
+        - [`MyComponent.style.js`](./components/my-component/style.s.md)
+        - [`index.js`](./components/my-component/index.s.md)
+    - `/redux`
+      - [`rootSaga.js`](./redux/root-saga.js)
+      - [`rootReducer.js`](./redux/root-reducer.js)
+      - `store.js`
+      - `/myModule`
+        - `actionTypes.js`
+        - `actions.js`
+        - `reducer.js`
+        - `api.js`
+        - `modelize.js`
+        - `normalize.js`
```

Comment:
je ne les connais pas ceux là

Reply:
@Nhacsam ils sont si beaux https://github.com/theodo/bnp-investo/tree/integration/src/redux/Funds

Reply:
@xavierlefevre no access to the link ...

Reply:
me too :'(